ducc/lib/conversion.go

```diff
@@ -74,7 +74,7 @@ func ConvertWishFlat(wish WishFriendly) error {
 		l.Log().WithFields(log.Fields{
 			"image":                  inputImage.GetSimpleName(),
 			"public path":            completePubSymPath,
-			"err stats pubblic path": errPub,
+			"err stats public path": errPub,
```

Comment:
The go compiler will hate me here, so this block needs to have its whitespace fixed